backend/src/controllers/renter.ts

```diff
@@ -0,0 +1,40 @@
+import { RequestHandler } from "express";
+
+import { asyncHandler } from "./wrappers";
+
+import { createRenterCandidate, getRenterCandidates } from "@/services/renter";
+
+export const getRenterCandidatesHandler: RequestHandler = asyncHandler(async (_req, res, _next) => {
+  const renters = await getRenterCandidates();
+
+  res.status(200).json(renters);
+});
+
+type CreateRenterCandidateRequestBody = {
```

Comment:
Would this have to be updated too with the new fields?

Reply:
Updated!